compiler/ast/reports.nim

```diff
@@ -343,18 +343,18 @@ type
   ReportSet* = object
     ids: PackedSet[uint32]
 
-func incl*(s: var ReportSet, id: ReportId) = s.ids.incl uint32(id)
-func contains*(s: var ReportSet, id: ReportId): bool = s.ids.contains uint32(id)
+func incl*(s: var ReportSet, id: NodeId) = s.ids.incl uint32(id)
```

Comment:
Is my understanding correct in that `ReportList` and the associated procedures are obsolete now?

Reply:
The only use it has at this point is to track duplicate reporting, but yeah list is pointless.

Reply:
I've removed `ReportList` type and the `reports` field from `MsgConfig`.